### PR TITLE
Responsive components

### PR DESCRIPTION
**Pull request by victorhmp**
#### What is the purpose of this pull request?

Add CSS classes from VTEX Tachyons to make each component responsive to
different screen sizes.

#### What problem is this solving?

Broken layouts on different screen sizes.

#### Types of changes
- [  ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [  ] Breaking change (fix or feature that would cause existing functionality to change)
- [  ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/Header/Header.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ const Header = () => {
   return (
     <div className="h-75 pt7 sans-serif">
 
-      <p className="tc serious-black f2">
+      <p className="tc serious-black t-heading-1">
```
Comment (lbebber) [inline react/components/Header/Header.js]: aproveita e muda de `serious-black` pra `c-on-base`

Comment (lbebber) [inline react/components/Header/Header.js]:
```suggestion
       <p className="tc c-on-base t-heading-1">
```

File: react/components/Header/Warnings.js [hunk 2]
```diff
@@ -5,8 +5,8 @@ const Info = () => {
   const twoOrders = true
   return (
     <Fragment>
-      <p><IconWarning color="#FFB100" /><strong className="pl3">Informações importantes:</strong></p>
-      <ul className="mv9 list ml0 pl0">
+      <p className="t-heading-6"><IconWarning color="#FFB100" /><strong className="pl3">Informações importantes:</strong></p>
```
Comment (lbebber) [inline react/components/Header/Warnings.js]: pra adicionar o token de cor no icone tambem, ao inves de hardcoded:

```suggestion
      <p className="t-heading-6"><span className="c-warning"><IconWarning color="currentColor" /></span><strong className="pl3">Informações importantes:</strong></p>
```

Comment (lbebber) [reply react/components/Header/Warnings.js]: só ve a formataçao dessa linha (espaços etc) antes de aceitar essa sugestao

Comment (victorhmp) [reply react/components/Header/Warnings.js]: Feito!

**Pull request merged**